federation/plugin/pkg/admission/schedulingpolicy/admission_test.go

```diff
@@ -343,6 +343,7 @@ func newControllerWithTestServer(f func(w http.ResponseWriter, r *http.Request),
 	if err != nil {
 		return nil, err
 	}
+	defer file.Close()
```

Comment:
The defer call a few lines above (https://github.com/kubernetes/kubernetes/pull/48013/files#diff-a277f7e642ed73686876c214453fab77R340) removes the file entirely. Is it really required to close it in a defer call that just executes before remove?